ansible/roles/edu-sharing/vars/versions/7.0.0.yml

```diff
@@ -0,0 +1,20 @@
+---
+edu_sharing_archive_url: https://artifacts.edu-sharing.com/repository/maven-remote/org/edu_sharing/edu_sharing-projects-community-deploy-docker-compose/7.0.0/edu_sharing-projects-community-deploy-docker-compose-7.0.0-bin.zip
+
+# Enable/Disable migration from previous Edu-Sharing version
+# This must run once and then please make it false
+edu_sharing_enable_migration: true
+#  This will create backups for old versions, since previous backup are not compatible
+# Enable/disable backup backup for old versions
+edu_sharing_enable_backup: true
```

Comment:
Is not an "enable-backup-switch", it is "processing-the-backup-during-installation-immediately", correct? Why do we need this? For the migration? If so, please also include in automatic detection